### PR TITLE
:sparkles: Fetch filtered data via dates and keywords

### PR DESCRIPTION
**Pull request by nrmnqdds**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
+  "scripts": {
+    "start": "node -r dotenv/config scraper.js"
+  },
+  "dependencies": {
+    "@babel/node": "^7.22.10",
+    "dotenv": "^16.3.1",
+    "rettiwt-api": "^2.0.3"
```
Comment (Zyten) [inline package.json]: If not mistaken, you're also using the rettiwt-auth package.

"rettiwt-auth": "^1.2.0"

File: scraper.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    // Creating a new Rettiwt instance using the cookies
+    const rettiwt = new Rettiwt(credential.cookies);
+
+    rettiwt.tweet
+      .search(
+        {
+          fromUsers: ["askrapidkl"],
+          words: ["kesulitan", "maaf"],
+          startDate: new Date("Sun Aug 20 07:12:31 +0000 2023"), //startDate -1
+          endDate: new Date("Tue Aug 22 01:26:23 +0000 2023"), //endDate +1
```
Comment (Zyten) [inline scraper.js]: I suspect eventually, we'd want this to be configurable. But for starters lets make it pull tweets from the last 24 hours like so:

```
startDate: new Date(Date.now() - 86400000),
endDate: new Date(),
```

**Review by Zyten — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (nrmnqdds) [reply package.json]: oh yea i installed it globally i forgot, my bad

**Review by Zyten — APPROVED** [review]
Looks good to me - fixes #2

**Pull request merged**